### ISSUE TITLE
Empty Dates

### ISSUE THREAD

**Issue by rtk170** [rudotriton/scriptable-calendar-widget#11]
Hi, 

I have been using your script for a while and it’s great! 
I’m trying to edit the script so that the previous month dates show instead of empty days at the start of the month (when the 1st of month isn’t on a Sunday).

I have used `var date = new Date();
date.setDate(0); ` to get the last day of the previous month but I’m stuck on how to apply this to the ‘buildMonthVertical’ function - any advice would be appreciated!

Thanks!

**Comment by rudotriton** [comment]
I had to change how this script is developed, because it was becoming increasingly more difficult to support new features.

If you are wondering, showing dates from surrounding months is a bit more involved than that, because the number of dates that fit in the front changes based on the month. It also changes based on if you count your days starting from Sunday or Monday. And the same applies if you want to add dates from the next month. Since the scrips is also counting the events for every visible day, it needs to take into account those extra days now. However, I have added two settings to reflect that: `showPrevMonth` and `showNextMonth` so it is now possible to do something like this:

<p align="center" >
    <img width="400" alt="scriptable calendar" src ="https://user-images.githubusercontent.com/14181026/109416468-199b6800-79c7-11eb-81d7-0255208b894d.png">
</p>

I hope it is roughly what you were looking for.
